lightstep/resource_metric_condition.go

```diff
@@ -82,11 +82,30 @@ func resourceUnifiedCondition(conditionSchemaType ConditionSchemaType) *schema.R
 		resource.Schema["expression"] = getUnifiedAlertExpressionSchema()
 		resource.Schema["query"] = &schema.Schema{
 			Type:     schema.TypeList,
-			Required: true,
+			Optional: true,
 			Elem: &schema.Resource{
 				Schema: getUnifiedQuerySchemaMap(),
 			},
 		}
+		resource.Schema["composite_alert"] = &schema.Schema{
```

Comment:
I think it might be nice if we started adding descriptions to new terraform fields. Would you mind copying over the public api description?